### ISSUE TITLE
Missing Ages Tree portal

### ISSUE THREAD

**Issue by twijgers** [vinheim3/oracles-disasm#1]
I was playing a crossworld game and I ran into a softlock. What I did, was do D1 Seasons (after opening the path to the Seasons tree), followed by D1 and D2 Ages. I got the harp _without_ saving the Ages tree. Now, I'm done with D2A, and I need to go back to the present (whether it is to progress in Ages or to go back to Seasons).

However, the only time portal I can think of that is accessible to me at the moment, is the portal in front of the Ages Tree, which is missing (presumably because of me not saving the tree?). I walked up to her (in the past), but it seems like I should have talked to her in the present first to trigger the "save the tree" event. I think I'm properly soft-locked now.

I think a way to fix this would be to always have the Ages tree be disappeared in the present, if this is really the cause of the issue. Either way, my portal isn't here right now.

**Comment by vinheim3** [comment]
This is caused by having the seed satchel in seasons. In ages, a number of events are prevented from happening once you have the satchel, so will need to look at reenabling these events safely.

I don't think there are other soft locks, so if you'd like to try some time later, just make sure to do the whole beginning past section of ages first, before getting the satchel in seasons

**Comment by twijgers** [comment]
Fair enough, thanks! I was just hoping I could skip the Ages tree completely... :P
